iothub_client/samples/pnp/pnp_simple_thermostat/pnp_simple_thermostat.c

```diff
@@ -222,88 +222,81 @@ static bool BuildMaxMinCommandResponse(unsigned char** response, size_t* respons
 // SetEmptyCommandResponse sets the response to be an empty JSON.  IoT Hub needs
 // legal JSON, regardless of error status, so if command implementation did not set this do so here.
 //
-static void SetEmptyCommandResponse(unsigned char** response, size_t* responseSize, int* result)
+static void SetEmptyCommandResponse(IOTHUB_CLIENT_COMMAND_RESPONSE* commandResponse)
 {
-    if ((*response = calloc(1, g_JSONEmptySize)) == NULL)
+    if ((commandResponse->payload = calloc(1, g_JSONEmptySize)) == NULL)
     {
         LogError("Unable to allocate empty JSON response");
```

Comment:
nit: c sdk style logging -- with or without '.' at end of sentence?  discussion in emb c sdk was to have the period.

Reply:
This repo isn't consistent (of course :().  

But a quick PowerShell (and I think Regex is correct, eyeballing it the # seems in ballpark) shows about a 9:1 ratio of no "." versus yes "."  So following the more common suggestion suggests no '.".

```script
C:\iot2\azure-iot-sdk-c\iothub_client> $a = gci -Recurse |Select-String "LogError"
C:\iot2\azure-iot-sdk-c\iothub_client> $a.count
2534
C:\iot2\azure-iot-sdk-c\iothub_client> ($a |Select-String '\."').count
245
```